### ISSUE TITLE
I'd love to try it, but....

### ISSUE THREAD

**Issue by NoSloppy** [ryryog25/ProffieConfig#1]
.
![Screenshot 2023-10-30 at 5 16 41 PM](https://github.com/ryryog25/ProffieConfig/assets/53964195/e7138266-cb1d-4980-96cf-c04c45a0673c)

**Comment by ryryog25** [comment]
Oh joy :)

First things first... is this on an Apple silicon mac?

**Comment by NoSloppy** [comment]
Nope. Intel.

**Comment by ryryog25** [comment]
Since I was being a bit overzealous with the .app... If you open the contents and run the ProffieConfig executable itself, any dice?

**Comment by NoSloppy** [comment]
There's no file name extension.
Double clicking it, it doesn't like the unknown developer.
Right click>open anyway, it opens in textedit.

cd to the MacOS directory and do `./ProffieConfig` no permissions.
chmod 777 then try running again:
```
dyld[73052]: Library not loaded: /usr/local/lib/libwx_osx_cocoau_xrc-3.3.dylib
  Referenced from: <DD7DD711-AD95-3B90-A287-DDCA1C7876C1> /Users/bconner/Downloads/macOS/ProffieConfig.app/Contents/MacOS/ProffieConfig
  Reason: tried: '/usr/local/lib/libwx_osx_cocoau_xrc-3.3.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/lib/libwx_osx_cocoau_xrc-3.3.dylib' (no such file), '/usr/local/lib/libwx_osx_cocoau_xrc-3.3.dylib' (no such file), '/usr/lib/libwx_osx_cocoau_xrc-3.3.dylib' (no such file, not in dyld cache)
Abort trap: 6

```

**Comment by ryryog25** [comment]
Ah, ok. I think I see what's going on... it compiled with dynamically linked libraries instead of statically linked, so if you don't have wx on your computer it won't work... let me see if I can fix that.

**Comment by NoSloppy** [comment]
No, the package has `.app`.
The executable has no extension.

**Comment by ryryog25** [comment]
Oh, I gotcha. yeah, it shouldn't

**Comment by ryryog25** [comment]
Ok, recompiled with static linked libraries, try this guy out. (A significantly heavier binary now that it has (hopefully) everything it needs, lol)
[ProffieConfig.app.zip](https://github.com/ryryog25/ProffieConfig/files/13210183/ProffieConfig.app.zip)

**Comment by ryryog25** [comment]
This issue has been resolved in v1.1

If you see it persisting, please reopen the issue.
